### PR TITLE
Fix: Don't Override `robots.txt` for Other Domains

### PR DESCRIPTION
**Pull request by ItzNotABug**
## What does this PR do?

This PR allows using custom `robots.txt` on function domains.

## Test Plan

Manual -

Robots.txt

Screenshot 2024-05-23 at 2 58 16 PM

Screenshot 2024-05-23 at 2 58 22 PM

Humans.txt

Screenshot 2024-05-23 at 3 57 24 PM

Screenshot 2024-05-23 at 3 57 29 PM

## Related PRs and Issues

N/A

## Checklist

- [x] Have you read the [Contributing Guidelines on issues](https://github.com/appwrite/appwrite/blob/master/CONTRIBUTING.md)?
- [x] If the PR includes a change to an API's metadata (desc, label, params, etc.), does it also include updated API specs and example docs?

### CODE REVIEW

**Review by Meldiron — APPROVED** [review]
Code looks good 🙌
Not 100% sure about approach, let's wait for 1 more approving review before merging

**Review by christyjacob4 — APPROVED** [review]

**Pull request merged**
